net.lr.example.kubernetes/src/main/java/net/lr/ds/hello/AliveCheck.java

```diff
@@ -23,17 +23,17 @@
 	
 	@Override
 	public String getName() {
-		return "Switchable alive check";
+		return "Toggable alive check";
```

Comment:
Not sure if Toggable is an engish word https://www.englishforums.com/English/Toggable/bvxvb/post.htm

Reply:
yeah, tou're right. There is "togglable" or "toggleable", but they sound weird. I can make that back to "switchable".